app/jobs/scheduled/clean_up_uploads.rb

```diff
@@ -22,11 +22,13 @@ def execute(args)
         .joins("LEFT JOIN user_avatars ua ON (ua.gravatar_upload_id = uploads.id OR ua.custom_upload_id = uploads.id)")
         .joins("LEFT JOIN user_profiles up ON up.profile_background = uploads.url OR up.card_background = uploads.url")
         .joins("LEFT JOIN categories c ON c.uploaded_logo_id = uploads.id OR c.uploaded_background_id = uploads.id")
+        .joins("LEFT JOIN custom_emojis ce ON ce.upload_id = uploads.id")
```

Comment:
Just a note for myself: we gotta stop adding more `joins` to that query and instead add a new column to the `uploads` table in order to store where the upload is being used (or if it's being used).